org.eclipse.wb.core/src/org/eclipse/wb/internal/core/utils/dialogfields/LayoutDialogFieldGroup.java

```diff
@@ -0,0 +1,300 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2011 IBM Corporation and others.

```

Comment:
Where does this class come from?

Reply:
Hi Wim. I'll try and contact the the colleague who originally added this. It's been a couple of years since then so he might not remember